core/menuitem.js

```diff
@@ -223,12 +223,12 @@ Blockly.MenuItem.prototype.setChecked = function(checked) {
 Blockly.MenuItem.prototype.setHighlighted = function(highlight) {
   this.highlight_ = highlight;
 
-  var el = this.getElement();
+  const el = this.getElement();
   if (el && this.isEnabled()) {
     // goog-menuitem-highlight is deprecated, use blocklyMenuItemHighlight.
     // May 2020.
-    var name = 'blocklyMenuItemHighlight';
-    var nameDep = 'goog-menuitem-highlight';
+    const name = 'blocklyMenuItemHighlight';
+    const nameDep = 'goog-menuitem-highlight';
```

Comment:
Side node: These look like they should be top-level constants.

Reply:
(Ignorant noob observation:) I'd sort of expect them to be imported from a master list of CSS class names or something like that.